### PR TITLE
Add example

### PR DESCRIPTION
**Pull request by yolile**
ref https://github.com/open-contracting/ocds-extensions/issues/120

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,43 @@ However, in cases where it is important to track documents on a milestone-by-mil
 
 Depending on the nature of the documents, publishers should consider duplicating information in the parent sections documents block also.
 
+## Worked Example
```
Comment (jpmckinney) [inline README.md]: Just use "Example" in extensions.

File: README.md [hunk 2]
```diff
@@ -14,6 +14,43 @@ However, in cases where it is important to track documents on a milestone-by-mil
...
+
+During the implementation of a contract, the buyer set different milestones that the supplier must met. Each of these milestone could has an associated document that offers more details about the met milestone, for example a consultancy report
```
Comment (jpmckinney) [inline README.md]: Please run through a grammar/spell checker (like Grammarly, etc.). Also, avoid normative words when describing an example (see [synonyms](https://tools.ietf.org/html/draft-hansen-nonkeywords-non2119-04#page-3)):

> During the implementation of a contract, the buyer sets milestones that the supplier needs to meet. Each of these milestones can have an associated document that offers more details about the met milestone: for example, a consultancy report:

Comment (yolile) [reply README.md]: sorry done!

File: README.md (hunk 1 above)
Comment (yolile) [reply README.md]: Sorry, I followed the [lot extension](https://github.com/open-contracting-extensions/ocds_lots_extension) pattern

**Pull request merged**